.sonarcloud.properties

```diff
@@ -1,2 +1,5 @@
 #we need to explicitly exclude them as they're not relevant to the source code
-sonar.exclusions=packages/glee/examples/**/*,packages/glee/docs/**/*
\ No newline at end of file
+sonar.exclusions=packages/glee/examples/**/*,packages/glee/docs/**/*,packages/gleequore/docs/**/*
+
+# Ignore all duplicated code as it will be removed once we replace Glee's core with GleeQuore
+sonar.cpd.exclusions=packages/glee/src/adapters/**/*,packages/*-adapter/**/*,packages/shared-utils/**/*,packages/gleequore/**/*
```

Comment:
❗ Attention: this is a temporary but important change. Since we're in an intermediate step where lots of code is duplicated, Sonar is complaining. I'm adding this here for now but should be removed once we replace Glee's core with GleeQuore.